google/pubsub/v1/pubsub_gapic.yaml

```diff
@@ -374,11 +381,16 @@ interfaces:
     - DEADLINE_EXCEEDED
   - name: one_plus_delivery
     retry_codes:
+    - ABORTED
+    - CANCELLED
+    - INTERNAL
```

Comment:
The comments by @jba on https://github.com/googleapis/googleapis/pull/283 concern me about retrying `INTERNAL`.